Panic when dumping a struct with a private map field

When dumping a struct that has a private field containing a map litter panics because the map key sort function calls .Interface() on an unexported value.
Repro case:
```go
package main

import (
	"github.com/sanity-io/litter"
)

type privateMap struct {
	private map[string]map[string]bool
}

func main() {
	litter.Options{
		HidePrivateFields: false,
		StripPackageNames: true,
	}.Dump(&privateMap{
		private: map[string]map[string]bool{
			"panics": {
				"panics": true,
			},
			"test": {
				"test": true,
			},
		},
	})
}
```

Results in error:
```
&privateMap{
  private: map[string]map[string]bool{
panic: reflect.Value.Interface: cannot return value obtained from unexported field or method
```
With the stack trace attributing this panic to dump.go:392

--- a/dump.go
+++ b/dump.go
@@ -425,5 +425,9 @@ func (s mapKeySorter) Swap(i, j int) {
 }
 
 func (s mapKeySorter) Less(i, j int) bool {
-	return fmt.Sprintf("%s", s.keys[i].Interface()) < fmt.Sprintf("%s", s.keys[j].Interface())
+	if s.keys[i].CanInterface() && s.keys[j].CanInterface() {
+		return fmt.Sprintf("%s", s.keys[i].Interface()) < fmt.Sprintf("%s", s.keys[j].Interface())
+	}
+	
+	return true
 }

